packages/flutter/lib/src/services/asset_vendor.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:async';
+
+import 'package:mojo/core.dart' as core;
+
+import 'asset_bundle.dart';
+import 'asset_resolver.dart';
+
+/// Configuration parameters for a particular AssetResolver.
+///
+/// Since AssetVendor uses configuration objects as cache keys, subclasses of
+/// AssetResolverConfig should override operator== and hashCode so that
+/// equivalent configurations appear to be equal. Config objects should also be
+/// cheap to construct.
+abstract class AssetResolverConfig {
+  const AssetResolverConfig();
+  AssetResolver createResolver();
+  operator==(AssetResolverConfig config);
+  int get hashCode;
```

Comment:
`int get hashCode` is redundant, the super class (`Object`) already defines that.